src/redux/actions/RocketsActions.js

```diff
@@ -0,0 +1,28 @@
+export const LOAD_ROCKETS = 'LOAD_ROCKETS';
+
+export const loadRockets = (payload) => ({
+  type: LOAD_ROCKETS,
+  payload,
+});
+
+async function fetchData() {
+  const response = await fetch('https://api.spacexdata.com/v3/rockets');
+  const data = await response.json();
+  return data;
+}
+
+export const displayRockets = () => async (dispatch) => {
+  const rocketsArr = await fetchData();
```

Comment:
Kindly consider renaming this function.
```suggestion
async function fetchRockets() {
  const response = await fetch('https://api.spacexdata.com/v3/rockets');
  const data = await response.json();
  return data;
}

export const displayRockets = () => async (dispatch) => {
  const rocketsArr = await fetchRockets();
```